profile-dotemacs.el

```diff
@@ -104,9 +107,12 @@ grayed out.")
 
 ;;;###autoload
 (defun profile-dotemacs ()
-  "Load `profile-dotemacs-file' and benchmark its sexps."
+  "Load the first file found from
+`profile-dotemacs-user-init-files' and benchmark its sexps."
```

Comment:
The first line of the docstring should be a self-contained sentence not extending farther than than 70 columns. More information can be put on subsequent lines.

Reply:
Thanks for the advice.